buffer/src/main/java/io/netty/buffer/EmptyByteBuf.java

```diff
@@ -96,6 +96,11 @@ public ByteBuf unwrap() {
     }
 
     @Override
+    public boolean isReadOnly() {
+        return true;
```

Comment:
@trustin I think this is not really correct as if this would be the case it would throw a `ReadOnlyBufferException` on write /set operations.